lang/th.js

```diff
@@ -0,0 +1,220 @@
+	"components.meter-mixin.fraction": "{x}/{y}",
+	"components.meter-mixin.fractionAria": "{x} จาก {y}",
+	"components.meter-mixin.progressIndicator": "ตัวบ่งชี้ความก้าวหน้า",
+	"components.more-less.less": "น้อยกว่า",
+	"components.more-less.more": "เพิ่มเติม",
+	"components.object-property-list.item-placeholder-text": "รายการตัวแทน",
+	"components.overflow-group.moreActions": "การดำเนินการเพิ่มเติม",
+	"components.pageable.info":
+		`{count, plural,
+			other {{countFormatted} รายการ}
+		}`,
+	"components.pageable.info-with-total":
+		`{totalCount, plural,
+			other {{countFormatted} จาก {totalCountFormatted} รายการ}
+		}`,
+	"components.pager-load-more.action": "โหลดเพิ่มเติม",
+	"components.pager-load-more.action-with-page-size": "โหลดเพิ่ม {count} รายการ",
+	"components.pager-load-more.status-loading": "กำลังโหลดรายการเพิ่มเติม",
+	"components.selection.action-max-hint":
+		`{count, plural,
+			other {ปิดใช้งานเมื่อเลือกมากกว่า {countFormatted} รายการ}
+		}`,
+	"components.selection.action-required-hint": "เลือกรายการที่จะดำเนินการนี้",
+	"components.selection.select-all": "เลือกทั้งหมด",
+	"components.selection.select-all-items": "เลือกทั้ง {count} รายการ",
+	"components.selection.selected": "{count} ที่เลือกแล้ว",
+	"components.selection.selected-plus": "{count}+ ที่เลือกแล้ว",
+	"components.selection-controls.label": "การดำเนินการสำหรับการเลือก",
+	"components.sort.label": "Sort",
+	"components.sort.text": "Sort: {selectedItemText}",
+	"components.switch.conditions": "ต้องตรงตามเงื่อนไข",
+	"components.switch.hidden": "ซ่อนอยู่",
+	"components.switch.visible": "มองเห็นได้",
+	"components.switch.visibleWithPeriod": "มองเห็นได้",
+	"components.table-col-sort-button.addSortOrder": "เลือกเพื่อเพิ่มลำดับการเรียง",
+	"components.table-col-sort-button.changeSortOrder": "เลือกเพื่อเปลี่ยนลำดับการเรียง",
+	"components.table-col-sort-button.title":
+		`{sourceType, select,
+			dates {{direction, select,
+				desc {เรียงจากใหม่ไปเก่า}
+				other {เรียงจากเก่าไปใหม่}
+			}}
+			numbers {{direction, select,
+				desc {เรียงจากสูงไปต่ำ}
+				other {เรียงจากต่ำไปสูง}
+			}}
+			words {{direction, select,
+				desc {เรียงจาก Z ไป A}
+				other {เรียงจาก A ไป Z}
+			}}
+			value {เรียงลำดับ {selectedMenuItemText}}
+			other {{direction, select,
+				desc {เรียงจากมากไปน้อย}
+				other {เรียงจากน้อยไปมาก}
+			}}
+		}`,
+	"components.table-controls.label": "การดำเนินการสำหรับตาราง",
+	"components.tabs.next": "เลื่อนไปข้างหน้า",
+	"components.tabs.previous": "เลื่อนย้อนกลับ",
+	"components.tag-list.clear": "คลิก กด backspace หรือกดคีย์ delete เพื่อลบรายการ {value}",
+	"components.tag-list.clear-all": "ล้างทั้งหมด",
+	"components.tag-list.cleared-all": "ลบรายการแท็กทั้งหมด",
+	"components.tag-list.cleared-item": "ลบรายการแท็ก {value} แล้ว",
+	"components.tag-list.interactive-label": "รายการแท็ก {count} รายการ",
+	"components.tag-list.num-hidden": "+ อีก {count} รายการ",
+	"components.tag-list.role-description":
```

Comment:
This term was added in July (after the export) so its translation will come in future update.